quantify-htseqcount.wdl

```diff
@@ -0,0 +1,20 @@
+# This workflow seems a bit redundant, since it only calls the HTSeqCount task.
+# It is included to keep the same structure as with the others.
+
+import "tasks/htseq.wdl" as htseq
+
+workflow QuantifyHTSeqCount {
+    Array[File] bamFiles
+    String outputDir
+    String sample
+
+    call htseq.HTSeqCount as htSeqCount {
```

Comment:
htseq also need strand information, same comment as above does apply on this then.

Reply:
I thought strandedness for this one might be given through the `inputs.json`, like the references.

Reply:
well don't want user to have multiple values for the same thing, that is confusing